tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -18,14 +22,20 @@
 #include <tclap/CmdLine.h>
 #include <tclap/ValueArg.h>
 
+#include <string>
 #include <iostream>
 #include <map>
 #include <unordered_set>
 
+using nlohmann::json;
 
 using namespace TCLAP;
 using namespace realdds;
 
+//Pointer to the context created in main, for access from other functions
+//Need to create a context only once or new context will create a dds_device_watcher and discover
+//devices we published as new DDS devices
+rs2::context * main_ctx = nullptr;
```

Comment:
Not sure I understand this.
I fixed a bug in the python that created a context automatically and made DDS usage impossible. Is this the problem you're trying to get around?

Reply:
Removed.

Was using this in functions that moved to `lrs-device-controller` and needed to use the context